### PR TITLE
Add swiss_schemaorg profile

### PR DESCRIPTION
**Pull request by metaodi**
This PR relies on https://github.com/ckan/ckanext-dcat/pull/108, it defines the mapping for DCAT-AP Switzerland to http://schema.org/Dataset

### CODE REVIEW

File: ckanext/switzerland/dcat/profiles.py [hunk 1]
```diff
@@ -46,12 +48,69 @@
...
 
 slug_id_pattern = re.compile('[^/]+(?=/$|$)')
 
 
-class SwissDCATAPProfile(RDFProfile):
+class MultiLangProfile(RDFProfile):
+    def _add_multilang_value(self, subject, predicate, dataset_key=None, dataset_dict=None, multilang_values=None):  # noqa
+        if not multilang_values and dataset_dict and dataset_key:
+            multilang_values = dataset_dict.get(dataset_key)
+        if multilang_values:
```
Comment (stefina) [inline ckanext/switzerland/dcat/profiles.py]: not sure if it makes sense to provide a fallback and just add the subject and predicate to the graph as a simple Literal if that is not misleading?

Comment (stefina) [reply ckanext/switzerland/dcat/profiles.py]: Pretty much, what is being done in https://github.com/opendata-swiss/ckanext-switzerland/pull/177/files#diff-3ec9836d38fb4719275ebd0bd7cd202bR74

File: ckanext/switzerland/dcat/profiles.py [hunk 2]
```diff
@@ -611,3 +652,125 @@ def graph_from_dataset(self, dataset_dict, dataset_ref):  # noqa
...
+                    SCHEMA.name,
+                    multilang_values=publisher_name
+                )
+            else:
+                g.add((publisher_details, SCHEMA.name, Literal(publisher_name)))  # noqa
+
+            contact_point = BNode()
+            self.g.add((publisher_details, SCHEMA.contactPoint, contact_point))
+
+            self.g.add((contact_point, SCHEMA.contactType, Literal('customer service')))  # noqa
```
Comment (stefina) [inline ckanext/switzerland/dcat/profiles.py]: is this intended to be 'customer service' ?

File: ckanext/switzerland/dcat/profiles.py [hunk 3]
```diff
@@ -611,3 +652,125 @@ def graph_from_dataset(self, dataset_dict, dataset_ref):  # noqa
...
+                ('publisher_email', SCHEMA.email, ['contact_email', 'maintainer_email', 'author_email'], Literal),  # noqa
+                ('publisher_name', SCHEMA.name, ['contact_name', 'maintainer', 'author'], Literal),  # noqa
+            ]
+
+            self._add_triples_from_dict(dataset_dict, contact_point, items)
+
+    def _temporal_graph(self, dataset_ref, dataset_dict):
+        # schema.org temporalCoverage only allows to specify one temporal
+        # DCAT-AP Switzerland allows to specify multiple
+        # for the mapping we always use the first one
```
Comment (stefina) [inline ckanext/switzerland/dcat/profiles.py]: I hope that won't rain down on us in the future 👯

File: ckanext/switzerland/dcat/profiles.py (hunk 2 above)
Comment (metaodi) [reply ckanext/switzerland/dcat/profiles.py]: Yes

**Pull request merged**